.github/workflows/ci-tests.yml

```diff
@@ -166,6 +166,17 @@ jobs:
         run: task ci:reset
       - name: Run Cypress
         run: task ci:cypress
+      - name: Prepare Cypress output for Chromatic action
+        if: always()
```

Comment:
Why is this line necessary?

Reply:
The question is whether we want to run the visual regression test if the Cypress tests fail.

Even though tests fail they can still produce output so the visual regression tests can run. If they do developers know whether they are (also) facing visual regression problems.